Scrapers fail in setup-node step

Currently some runs fail during the "setup-node" step:

- https://github.com/openZH/covid_19/runs/563225628?check_suite_focus=true
- https://github.com/openZH/covid_19/runs/563208759?check_suite_focus=true


```
Premature close
Waiting 14 seconds before trying again
Premature close
Waiting 12 seconds before trying again
##[error]Premature close
```

Upstream issue: https://github.com/actions/setup-node/issues/132

--- a/.github/workflows/run_ag_scraper.yml
+++ b/.github/workflows/run_ag_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_ai_scraper.yml
+++ b/.github/workflows/run_ai_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_ar_scraper.yml
+++ b/.github/workflows/run_ar_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_be_scraper.yml
+++ b/.github/workflows/run_be_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_bl_scraper.yml
+++ b/.github/workflows/run_bl_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_bs_scraper.yml
+++ b/.github/workflows/run_bs_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_fr_scraper.yml
+++ b/.github/workflows/run_fr_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_gl_scraper.yml
+++ b/.github/workflows/run_gl_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_gr_scraper.yml
+++ b/.github/workflows/run_gr_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_ju_scraper.yml
+++ b/.github/workflows/run_ju_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_lu_scraper.yml
+++ b/.github/workflows/run_lu_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_nw_scraper.yml
+++ b/.github/workflows/run_nw_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_ow_scraper.yml
+++ b/.github/workflows/run_ow_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_sg_scraper.yml
+++ b/.github/workflows/run_sg_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_sh_scraper.yml
+++ b/.github/workflows/run_sh_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_so_scraper.yml
+++ b/.github/workflows/run_so_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_sz_scraper.yml
+++ b/.github/workflows/run_sz_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_tg_scraper.yml
+++ b/.github/workflows/run_tg_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_ti_scraper.yml
+++ b/.github/workflows/run_ti_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_ur_scraper.yml
+++ b/.github/workflows/run_ur_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_vd_scraper.yml
+++ b/.github/workflows/run_vd_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_vs_scraper.yml
+++ b/.github/workflows/run_vs_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

--- a/.github/workflows/run_zg_scraper.yml
+++ b/.github/workflows/run_zg_scraper.yml
@@ -18,10 +18,6 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
     - run: npm ci
     - name: Install dependencies
       run: |

